### ISSUE TITLE
RootBone position changes between Blender 2.91 and Blender 2.83 when animations are brought to UE4.26.

### ISSUE THREAD

**Issue by pto8913** [xavier150/Blender-For-UnrealEngine-Addons#86]
Environment :
Version of the add-on tested : 0.2.8.0
UE4 version : 4.26
Blender 2.91.2
Blender 2.83.12

Issue :
When using this add-on in Blender 2.83.12 and exporting an animation, RootBone comes to the specified position. (See image 1)
However, when I export the same animation in Blender 2.91.2, the RootBone comes to the same position as the hip bone. (See image 2)

Is there something wrong?
Or is this the correct specification?
image 1
![1](https://user-images.githubusercontent.com/39937503/109091199-ab04a300-7757-11eb-9e39-8fe3456b3c6b.png)
image 2
![2](https://user-images.githubusercontent.com/39937503/109091204-ac35d000-7757-11eb-8395-d9ce5968b805.png)

**Comment by xavier150** [comment]
Hello @pto8913 
That weird you can send me the file? I will try on my side. XavierLoux.loux@gmail.com

**Comment by pto8913** [comment]
One correction.
It turns out that this is not related to the Blender version.
This problem only happens with certain projects.

I tried the project (Blender 2.83.12) with add-on 0.2.6 and it worked fine.
It also worked fine in Blender 2.91.2.

Also, I tried Add-on 0.2.8.0 in another project and it worked fine.

So it seems to be happening only when I use add-on 0.2.8.0 with a specific file.

I don't know the cause of this problem, but it seems likely that it is my problem, so I will close it.

I'm sorry for upsetting you.
